Classes/CocoaLumberjack.swift

```diff
@@ -26,77 +26,75 @@ extension DDLogFlag {
     
     ///returns the log level, or the lowest equivalant.
     public func toLogLevel() -> DDLogLevel {
-        if let ourValid = DDLogLevel(rawValue: self.rawValue) {
+        if let ourValid = DDLogLevel(rawValue: rawValue) {
             return ourValid
         } else {
-            let logFlag:DDLogFlag = self
-            
-            if logFlag.contains(.Verbose) {
-                return .Verbose
-            } else if logFlag.contains(.Debug) {
-                return .Debug
-            } else if logFlag.contains(.Info) {
-                return .Info
-            } else if logFlag.contains(.Warning) {
-                return .Warning
-            } else if logFlag.contains(.Error) {
-                return .Error
+            if contains(.verbose) {
+                return .verbose
+            } else if contains(.debug) {
+                return .debug
+            } else if contains(.info) {
+                return .info
+            } else if contains(.warning) {
+                return .warning
+            } else if contains(.error) {
+                return .error
             } else {
-                return .Off
+                return .off
             }
         }
     }
 }
 
-public var defaultDebugLevel = DDLogLevel.Verbose
+public var defaultDebugLevel = DDLogLevel.verbose
 
 public func resetDefaultDebugLevel() {
-    defaultDebugLevel = DDLogLevel.Verbose
+    defaultDebugLevel = DDLogLevel.verbose
 }
 
-@available(*, deprecated, message="Use one of the DDLog*() functions if appropriate or call _DDLogMessage()")
-public func SwiftLogMacro(isAsynchronous: Bool, level: DDLogLevel, flag flg: DDLogFlag, context: Int = 0, file: StaticString = #file, function: StaticString = #function, line: UInt = #line, tag: AnyObject? = nil, @autoclosure string: () -> String, ddlog: DDLog = DDLog.sharedInstance()) {
+@available(*, deprecated, message: "Use one of the DDLog*() functions if appropriate or call _DDLogMessage()")
```

Comment:
I think this has been deprecated for a while, so it seems reasonable to me to remove it, though it doesn't really hurt to leave it there.